test/lib/clients/inst_productsources_test.rb

```diff
@@ -84,7 +85,10 @@ def list_of_repos
     end
 
     it "returns false if file is empty" do
-      expect(client.ParseListOfSources("/dev/zero", url)).to eq false
```

Comment:
Just pass "/dev/null", it is standard.

Reply:
dev null is not for reading, only for writting, not?

Reply:
also for reading, https://man7.org/linux/man-pages/man4/null.4.html